From 489a82f358ae8f00c4246bd9648043d8e97a508b Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 20 May 2020 08:43:43 +0300
Subject: [PATCH] [SW-11875] goya2: load arcboot_rom to SRAM

Currently arcboot_rom for onix platform is loaded in scratchpad SRAM.
Since we are loading preboot in scratchpad SRAM, change arcboot_rom to
load to SRAM.

Note that SRAM is comparably slow to ROM in execution but it should not
be an issue as we're not bringing up via code and loading of arcboot_rom
at SRAM locations is for debug purpose only.

Change-Id: I34e13717d9ba08f47fe133e24d697ad692940c6c
---

diff --git a/goya2_arc/onix_arcboot_rom.lds b/goya2_arc/onix_arcboot_rom.lds
index 908c04b..fdb13f0 100644
--- a/goya2_arc/onix_arcboot_rom.lds
+++ b/goya2_arc/onix_arcboot_rom.lds
@@ -9,5 +9,5 @@
 
 MEMORY
 {
-	SBROM	: ORIGIN = 0x1BFE1000, LENGTH = 60K
-}
\ No newline at end of file
+	SBROM	: ORIGIN = 0x10000000, LENGTH = 64K
+}
